register_types.cpp

```diff
@@ -4,7 +4,11 @@
 #include "core/hex_mesh.h"
 #include "core/mesh.h"
 #include "core/pent_mesh.h"
+#if SOTA_MODULE
+#include "core/object/class_db.h"
```

Comment:
Thank you for fix. Could you please move this include in `src/tal/godot_core.h` and include that file instead? There are includes for "SOTA_MODULE" in the bottom of file. I would like to encapsulate all things related to difference between gdextension and module in TAL. Also, see [doc](https://sota.readthedocs.io/en/latest/content/arch/tal.html)